src/kbc_modules/online_sev_kbc/keybroker.rs

```diff
@@ -0,0 +1,268 @@
+#[allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
Should this file should be committed? I think it is auto-generated.
and what about the other KBCs' auto-generated files?

Reply:
I'm working on "generating protobuf on demand", to reduce build time dependency.
So we have added an "gen-proto" feature to generate protobuf files on demand and commit those generated files.

Reply:
OK (maybe it should have gone to a separate PR to ease review).

Reply:
Can we add a note about this to the doc for the online_sev_kbc?

Reply:
Add one section to online_sev_kbc/README.md about how to update the protobuf file.